docs/api/api_blueprint.apib

```diff
@@ -4330,10 +4274,19 @@ Create a new tax using an event_id.
             {
               "data": {
                 "attributes": {
-                  "tax-name": "Service Tax",
-                  "tax-rate": "12",
-                  "tax-id": "1"
-                },
+                    "city": "example",
+                    "zip": "123456",
+                    "country": "IN",
```

Comment:
Validation for all the country fields is remaining and payment_countries only covers a few countries. I would prefer to do it for all the apis together in another issue, if that sounds okay with you :smile: